masonry/Cargo.toml

```diff
@@ -28,7 +28,7 @@ masonry_core.workspace = true
 masonry_testing = { workspace = true, optional = true }
 parley.workspace = true
 tracing = { workspace = true, features = ["default"] }
-ui-events.workspace = true
+ui-events = { workspace = true, features = ["kurbo"] }
```

Comment:
Is there a reason this feature isn't enabled at the workspace level? It seems like a poor DX for a contributor to try to use one of the methods this enables in Masonry Core, for it to then fail in CI.

Alternatively, can we also enable this feature in Masonry Core?

Reply:
(Indeed, it isn't even clear to me why `masonry` has this dependency, given that `masonry_core` re-exports `ui_events`)

Reply:
The re-export was probably something I was cleaning up before until I ran out of time.